platformio.ini

```diff
@@ -730,6 +730,34 @@ build_unflags = -std=gnu++11
 src_filter    = ${common.default_src_filter} +<src/HAL/HAL_SAMD51>
 debug_tool    = jlink
 
+#
+# RUMBA32
+#
+[env:rumba32_f446ve]
+platform      = ststm32
+framework     = arduino
+board         = rumba32_f446ve
+build_flags   = ${common.build_flags}
+  -DSTM32F4xx
+  -DARDUINO_RUMBA32_F446VE
+  -DARDUINO_ARCH_STM32
+  "-DBOARD_NAME=\"RUMBA32_F446VE\""
+  -DSTM32F446xx
+  -DUSBCON
+  -DUSBD_VID=0x0483
+  "-DUSB_MANUFACTURER=\"Unknown\""
+  "-DUSB_PRODUCT=\"RUMBA32_F446VE\""
+  -DHAL_PCD_MODULE_ENABLED
+  -DUSBD_USE_CDC
+  -DDISABLE_GENERIC_SERIALUSB
+  -DHAL_UART_MODULE_ENABLED
+  -Os
+lib_deps      = ${common.lib_deps}
+lib_ignore    = Adafruit NeoPixel
+src_filter    = ${common.default_src_filter} +<src/HAL/HAL_STM32> +<src/HAL/HAL_STM32_F4_F7> -<src/HAL/HAL_STM32_F4_F7/STM32F7>
```

Comment:
This should not be including HAL_STM32_F4_F7. It only needs HAL_STM32.

Reply:
Updated: e8f7036